pkg/auth/audit/audit_test.go

```diff
@@ -126,20 +139,188 @@ func Test_concealSensitiveData(t *testing.T) {
 			want:  machineDataWant,
 		},
 	}
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
+	for i := range tests {
+		test := tests[i]
+		a.Run(test.name, func() {
 			var want map[string]interface{}
-			if err := json.Unmarshal(tt.want, &want); err != nil {
-				t.Errorf("error unmarshaling: %v", err)
-			}
-			got := a.concealSensitiveData(tt.uri, tt.input)
+			err := json.Unmarshal(test.want, &want)
+			a.NoError(err, "failed to unmarshal")
+			got := logger.concealSensitiveData(test.uri, test.input)
 			var gotMap map[string]interface{}
-			if err := json.Unmarshal(got, &gotMap); err != nil {
-				t.Errorf("error unmarshaling: %v", err)
-			}
-			if !reflect.DeepEqual(want, gotMap) {
-				t.Errorf("concealSensitiveData() = %s, want %s", got, tt.want)
+			err = json.Unmarshal(got, &gotMap)
+			a.NoError(err, "failed to unmarshal")
+			a.Equal(want, gotMap, "concealSensitiveData() = %s, want %s", got, test.want)
+		})
+	}
+}
+func (a *AuditTest) TestCompression() {
```

Comment:
These tests are well-written. However, there doesn't seem to be any consideration for log-level. Given that you did change around the boolean logic, I'd like to see that somewhere if you don't mind.